tests/baselines/reference/nodeResolution7.errors.txt

```diff
@@ -0,0 +1,14 @@
+tests/cases/compiler/b.ts(1,20): error TS2306: File 'tests/cases/compiler/node_modules/a/index.d.ts' is not a module.
+
+
+==== tests/cases/compiler/b.ts (1 errors) ====
+    import y = require("a"); 
+                       ~~~
+!!! error TS2306: File 'index.d.ts' is not a module.

```

Comment:
It seems like this would be a good case of having the "Please contact the package author to update the package definition." message.